TheWarWithin/Priorities/MageArcane.simc

```diff
@@ -48,7 +48,7 @@ actions.cd_opener+=/arcane_blast,if=buff.presence_of_mind.up
 # Use Orb for charges if you have High Voltage, then evocation, then Missiles for Nether Precision, then Arcane Surge
 actions.cd_opener+=/arcane_orb,if=talent.high_voltage&buff.arcane_charge.stack<4,line_cd=10
 actions.cd_opener+=/evocation,if=cooldown.arcane_surge.remains<(gcd.max*3)&cooldown.touch_of_the_magi.remains<(gcd.max*5)
-actions.cd_opener+=/arcane_missiles,if=variable.opener&talent.nether_precision&buff.nether_precision.down,interrupt_if=tick_time>gcd.remains&buff.aether_attunement.down,interrupt_immediate=1,interrupt_global=1,chain=1,line_cd=30
+actions.cd_opener+=/arcane_missiles,if=prev_gcd.1.evocation|prev_gcd.1.arcane_orb,interrupt_if=tick_time>gcd.remains&buff.aether_attunement.down,interrupt_immediate=1,interrupt_global=1,chain=1,line_cd=30
```

Comment:
This is still not really going to work with the `line_cd` in place -- I recommend removing it since this will only fire if no Arcane Missiles have been cast in the past 30 seconds.  You probably need the `variable.opener` check so it doesn't fire at other times.

Reply:
Agree on line_cd, I think I had intended to remove it.

As for variable.opener on the missile, that is a condition requirement to even enter the `cd_opener` sublist in the first place, so it would be redundant.